src/Service/VendorService/TheMovieDatabase/Message/ApiSearchMessage.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/**
+ * @file
+ * Contains ApiSearchMessage that contains data about a search from the data
+ * well.
+ */
+
+namespace App\Service\VendorService\TheMovieDatabase\Message;
+
+use App\Service\MoreInfoService\Types\IdentifierType;
+
+/**
+ * Class ApiSearchMessage.
+ */
+class ApiSearchMessage implements \JsonSerializable
+{
+    private $vendorId;
+    private $identifierType;
+    private $pid;
+    private $title;
+    private $year;
+    private $originalYear;
+    private $director;
+
+    /**
+     * ApiSearchMessage constructor.
+     *
+     * @param int|null    $vendorId
+     *   The vendor id
+     * @param string|null $identifierType
+     *   The identifier type
+     * @param string|null $pid
+     *   The pid identifier
+     * @param array       $meta
```

Comment:
There is a lot of space here between type and variabel name?

Reply:
To align with above variables